lib/matplotlib/cbook.py

```diff
@@ -566,6 +566,18 @@ def process(self, s, *args, **kwargs):
                     self._remove_proxy(proxy)
 
 
+class EventEmitter(object):
+    def __init__(self):
+        super(EventEmitter, self).__init__()
```

Comment:
I have now read https://rhettinger.wordpress.com/2011/05/26/super-considered-super/ and scanned 
https://fuhm.net/super-harmful/.
What I think the two authors agree on is that if super is to be used, it must be used very carefully and consistently, and the usage strategy and scope needs to be clearly documented. So yes, if we are going the super route, then having something about it in the dev guide would be a good idea.